main.py

```diff
@@ -11,13 +11,24 @@
 @app.route('/v1/todos')
 @app.route('/v1/todos/')
 @app.route('/v1/todos/<status>')
-def list_all(status=ALL):
+def list_all(status=ALL) -> object:
     todos_list = todos_repo.list_all(status)
     return jsonify({
-            'items': [
-                x.to_dict() for x in todos_list
-            ],
-            'details': {
-                'activeToDos': sum(map(lambda x: x.has_status(Status.ACTIVE.value), todos_list))
-            }
-        })
+        'items': [
+            x.to_dict() for x in todos_list
+        ],
+        'details': {
+            'activeToDos': sum(map(lambda x: x.has_status(Status.ACTIVE.value), todos_list))
```

Comment:
What if the todos_list contains only inactive todos when also active todos exist?